schema.sql

```diff
@@ -0,0 +1,59 @@
+CREATE DATABASE catalog;
+
+CREATE TABLE
+    genres (
+        id SERIAL PRIMARY KEY,
+        name VARCHAR(100) NOT NULL
+    );
+
+CREATE TABLE
+    music_albums (
+        id SERIAL PRIMARY KEY,
+        title VARCHAR(100) NOT NULL,
+        genre_id INTEGER NOT NULL,
+        author_id INTEGER NOT NULL,
+        label_id INTEGER NOT NULL,
+        publish_date DATE NOT NULL,
+        archived BOOLEAN NOT NULL,
+        on_spotify BOOLEAN NOT NULL,
+        FOREIGN KEY (genre_id) REFERENCES genre (id) ON DELETE CASCADE
```

Comment:
- [ ] there is a typo in this line which stop the creation of the tables please make sure to  fix it by adding s as in the example so that It can create all the tables without probs
- [ ] optional - I suggest indexes for all the foreign keys in order to increase the performance when loading the data and searching 
``` suggestion
        FOREIGN KEY (genre_id) REFERENCES genres (id) ON DELETE CASCADE
```